indexing-service/src/main/java/org/apache/druid/indexing/common/task/AbstractBatchIndexTask.java

```diff
@@ -965,6 +990,35 @@ protected TaskReport.ReportMap buildIngestionStatsReport(
     );
   }
 
+  /**
+   * Builds a singleton map with {@link IngestionStatsAndErrorsTaskReport#REPORT_KEY}
+   * as key and an {@link IngestionStatsAndErrorsTaskReport} for this task as value.
+   */
+  protected TaskReport.ReportMap buildIngestionStatsReportWithoutContext(
```

Comment:
same comment, let's call this method `buildIngestionStatsReport()` and the older one `buildIngestionStatsAndContextReport()`.

Reply:
Done